src/frontend/src/tests/lib/components/convert/ConvertModal.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { BTC_MAINNET_TOKEN } from '$env/tokens.btc.env';
+import { ICP_TOKEN } from '$env/tokens.env';
+import ConvertModal from '$lib/components/convert/ConvertModal.svelte';
+import { CONVERT_CONTEXT_KEY } from '$lib/stores/convert.store';
+import en from '$tests/mocks/i18n.mock';
+import { fireEvent, render } from '@testing-library/svelte';
+import { readable } from 'svelte/store';
+
+describe('ConvertModal', () => {
+	const mockContext = () =>
+		new Map([
+			[
+				CONVERT_CONTEXT_KEY,
+				{
+					sourceToken: readable(BTC_MAINNET_TOKEN),
+					destinationToken: readable(ICP_TOKEN)
+				}
+			]
+		]);
+
+	it('should display correct modal title after navigating between steps', async () => {
```

Comment:
We have tests for all children components (where it was applicable), so in case of `ConvertModal`, I couldn't come up with some additional test suites that have not been covered yet. Also, Svelte doesn't really provide us with the possibility of checking component internal vars. IMO, all further tests belong to the e2e level.